chrome_launcher.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# Download Extension
+extension_setup() {
+  ps_analysis_tool_version=v0.3.0
```

Comment:
@MiteshShah 0.3.1 has been released, can we please change this to 0.3.1?

Reply:
on it!

Reply:
done